accepted/2021/statics-in-interfaces/README.md

```diff
@@ -789,90 +814,129 @@ namespace System
         static abstract TSelf TrailingZeroCount(TSelf value);
     }
 
-    public interface IFloatingPoint<TSelf>
-        : ISignedNumber<TSelf>
-        where TSelf : IFloatingPoint<TSelf>
+    public interface IExponentialFunctions<TSelf>
```

Comment:
These groupings are similar to the groupings in other languages and in language specs. They are still somewhat IEEE centric, but overall cover the core function + the common specializations for binary and decimal based numbers.

If DIMs are supported before .NET 7 ships, most of these functions could have a default implementation that does the "naive" thing.